pkg/security/resolvers/process/resolver.go

```diff
@@ -544,7 +544,7 @@ func (p *Resolver) insertEntry(entry, prev *model.ProcessCacheEntry, origin proc
 		prev.Release()
 	}
 
-	if p.cgroupResolver != nil {
+	if p.cgroupResolver != nil && entry.ContainerID != "" {
```

Comment:
Are you sure you want to filter out the host ? I know we have another filter later on to filter out the host, so maybe we should keep just one fail safe ?